4788/src/main/cpp/Robot.cpp

```diff
@@ -5,45 +5,35 @@ using namespace wml;
 
 using hand = frc::XboxController::JoystickHand;
 
-double currentTimeStamp;
+double currentTime;
 double lastTimeStamp;
 double dt; //stands for delta time 
 
 //add other variables here
-double SparkSpeed;
-double TalonSpeed;
+double sparkSpeed;
+double talonSpeed;
+double deadzone = 0.1;
 
 // Robot Logic
 void Robot::RobotInit() {
-	
 	dt = frc::Timer::GetFPGATimestamp() - lastTimeStamp;
```

Comment:
What's with these lines? Init happens before the rest of the program. And it happens only once. If you want a default value for dt and lastTimeStamp just set them as 0 when you declare them